components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -275,13 +280,7 @@ class ContributionFlowSuccess extends React.Component {
                     />
                   </P>
                 </Box>
-                {isEmbed ? (
-                  <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={false} />
-                ) : (
-                  <StyledLink as={Link} color="black.800" href={getCollectivePageRoute(order.toAccount)}>
-                    <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={false} />
-                  </StyledLink>
-                )}
+                <ContributorCardWithTier width={250} height={380} contribution={order} my={2} useLink={false} />
```

Comment:
link conflicts with button click target for now, but can be reverted if the [MVP design is followed and the button is separate](https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=16390%3A237486)